STM32F405RGTx/Core/Src/bms_entry.cpp

```diff
@@ -53,7 +49,7 @@ void cell_disable() {
 }
 
 void hard_fault_state_trans() {
```

Comment:
These functions should be prefixed by an `_`, since they're private to the error handler